--- kernel/drivers/net/can/usb/gs_usb.c	2026-04-21 15:41:09.719642541 +0800
+++ kernel/drivers/net/can/usb/gs_usb.c	2026-04-21 15:25:44.482626862 +0800
@@ -278,9 +278,9 @@
 /* Only launch a max of GS_MAX_RX_URBS usb requests at a time. */
 #define GS_MAX_RX_URBS 30
 /* Maximum number of interfaces the driver supports per device.
- * Current hardware only supports 3 interfaces. The future may vary.
+ * Current hardware only supports 4 interfaces. The future may vary.
  */
-#define GS_MAX_INTF 3
+#define GS_MAX_INTF 4
 
 struct gs_tx_context {
 	struct gs_can *dev;
